.env.tpl

```diff
@@ -1,23 +1,23 @@
-_AIRFLOW_DB_UPGRADE="true"
-_AIRFLOW_WWW_USER_CREATE="true"
+_AIRFLOW_DB_UPGRADE=${_AIRFLOW_DB_UPGRADE:-True}
+_AIRFLOW_WWW_USER_CREATE=${_AIRFLOW_WWW_USER_CREATE:-True}
 _AIRFLOW_WWW_USER_USERNAME=${_AIRFLOW_WWW_USER_USERNAME}
 _AIRFLOW_WWW_USER_PASSWORD=${_AIRFLOW_WWW_USER_PASSWORD}
 _AIRFLOW_WWW_USER_EMAIL=${_AIRFLOW_WWW_USER_EMAIL}
 _AIRFLOW_WWW_USER_FIRST_NAME=${_AIRFLOW_WWW_USER_FIRST_NAME}
 _AIRFLOW_WWW_USER_LAST_NAME=${_AIRFLOW_WWW_USER_LAST_NAME}
 _PIP_ADDITIONAL_REQUIREMENTS=${_PIP_ADDITIONAL_REQUIREMENTS}
-AIRFLOW__API__AUTH_BACKENDS="airflow.api.auth.backend.basic_auth"
+AIRFLOW__API__AUTH_BACKENDS=${AIRFLOW__API__AUTH_BACKENDS:-"airflow.api.auth.backend.basic_auth"}
 AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION=${AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION}
 AIRFLOW__CORE__FERNET_KEY=${AIRFLOW__CORE__FERNET_KEY}
 AIRFLOW__DATABASE__SQL_ALCHEMY_CONN=${AIRFLOW__DATABASE__SQL_ALCHEMY_CONN}
 AIRFLOW__WEBSERVER__SECRET_KEY=${AIRFLOW__WEBSERVER__SECRET_KEY}
-AIRFLOW__SMTP__SMTP_HOST=smtp.sendgrid.net
-AIRFLOW__SMTP__SMTP_STARTTLS=False
-AIRFLOW__SMTP__SMTP_SSL=False
-AIRFLOW__SMTP__SMTP_USER=apikey
-AIRFLOW__SMTP__SMTP_PASSWORD=<generated-api-key>
-AIRFLOW__SMTP__SMTP_PORT=587
-AIRFLOW__SMTP__SMTP_MAIL_FROM=<your-from-email>
+AIRFLOW__SMTP__SMTP_HOST=${AIRFLOW__SMTP__SMTP_HOST:-"smtp.sendgrid.net"}
```

Comment:
Hey @luabida , could you remove the defaul value from here. I think we don't need to suggest any provider here.

Reply:
Done!